pages/index.js

```diff
@@ -62,9 +76,33 @@ export default function HomePage({
             onChangeMode={handleChangeMode}
           />
         )}
-
+        <StyledFilterBox>
+          <label htmlFor="filter">Filter: </label>
+          <StyledFilterSelect
+            name="filter"
+            id="filter"
+            onChange={handleFilterChange}
+            value={filterCategory}
+          >
+            <option value="">---Choose a category---</option>
+            {categories.map((category) => {
+              return (
+                <option key={category} value={category}>
+                  {category}
+                </option>
+              );
+            })}
+          </StyledFilterSelect>
+          <StyledResetButton
+            onClick={() => {
+              setFilterCategory("");
+            }}
```

Comment:
being very picky here, you created a handler function for setting the category, it would be consistent to create a `handleFilterReset` function which does this.

Reply:
You being picky, gives me an opportunity to learn :) I implemented this.